app/internal/notes/notes.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from typing import Any, Dict, List
+
+from fastapi import HTTPException, status
+from sqlalchemy.orm import Session
+
+from app.database.models import Note
+from app.database.schemas import NoteSchema
+
+
+async def create(session: Session, payload: NoteSchema) -> int:
+    note = Note(title=payload.title, description=payload.description)
+    session.add(note)
+    session.commit()
+    session.refresh(note)
+    return note.id
+
+
+async def view(session: Session, id: int) -> Note:
+    note = session.query(Note).filter_by(id=id).first()
+    if not note:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=f"Note with id {id} not found",
+        )
+    return note
+
+
+async def get_all(
+    session: Session, skip: int = 0, limit: int = 100
+) -> List[Note]:
+    return session.query(Note).offset(skip).limit(limit).all()
+
+
+async def update(request: NoteSchema, session: Session, id: int) -> str:
+    note = session.query(Note).filter_by(id=id)
+    if not note.first():
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=f"Note with id {id} not found",
+        )
+    if request.timestamp is None:
+        request.timestamp = datetime.utcnow()
+    note.update(request)
+    session.commit()
+    return "updated"
+
+
+async def delete(session: Session, id: int) -> str:
+    note = session.query(Note).filter_by(id=id)
+    if not note.first():
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=f"Note with id {id} not found",
+        )
+    note.delete(synchronize_session=False)
+    session.commit()
+    return "deleted"
```

Comment:
return a boolen True. No need for text here.

Reply:
I think it is a nice addition to the swagger response instead of "True". Do you see it as a must change?
![image](https://user-images.githubusercontent.com/32178925/109183024-33a23400-7796-11eb-9ecf-62b59235279d.png)

Reply:
Prefer returning json: `{"success": True, "action": "deleted"}`